libs/hmpb/render-backend/package.json

```diff
@@ -47,7 +47,7 @@
     "js-sha256": "^0.9.0",
     "path": "^0.12.7",
     "pdf-lib": "^1.17.1",
-    "pdfkit": "^0.13.0",
+    "pdfkit": "^0.14.0",
```

Comment:
I don't think we actually use pdfkit anymore (pretty sure it got removed when I implemented the new HMPB render stack, but I forgot to clean up)